Client/Utils/SearchTermGroupsUtility.swift

```diff
@@ -7,7 +7,7 @@ import Shared
 import Storage
 import MozillaAppServices
 
-class SearchTermGroupsManager {
+class SearchTermGroupsUtility {
```

Comment:
This class doesn't manage anything. It is a utility to do something. Renamed for clarity.